section1/practice_exercises/ex5.rb

```diff
@@ -0,0 +1,17 @@
+name = 'Zed A. Shaw'
+age = 35 # not a lie in 2009
+height = 74 * 2.54 # cm
+weight = 180 / 2.2 # kg
+eyes = 'Blue'
+teeth = 'White'
+hair = 'Brown'
+
+puts "Let's talk about #{name}."
+puts "He's #{height} cm tall."
+puts "He's #{weight} kg heavy."
+puts "Actually that's not too heavy."
+puts "He's got #{eyes} eyes and #{hair} hair."
+puts "His teeth are usually #{teeth} depending on the coffee."
+
+# this line is tricky, try to get it exactly right
+puts "If I add #{age}, #{height}, and #{weight} I get #{age + height + weight}."
```

Comment:
:)

Challenge yourself: Your current `height` and `weight` are very long. How would you limit the number of decimals to just 2 (the hundreths place)